detox/e2e/test/products/channels/messaging/pin_and_unpin_message.e2e.ts

```
@@ -61,12 +58,33 @@ async function openChannelPostOptionsForPin(postId: string, message: string) {
             return;
         } catch {
             if (attempt === 3) {
-                throw new Error(`Post options did not appear for "${message}" after ${attempt} attempts`);
+                await ChannelScreen.openPostOptionsFor(postId, message);
```

ESLint

[error] Unexpected `await` inside a loop.

(no-await-in-loop)